app/views/blogs/index.html.haml

```diff
@@ -0,0 +1,26 @@
+= link_to "Back", :back
+= button_to "Create Blog", new_blog_path, :method => :get
+
+- @blog.each do |blogs|
+  - if blogs.user_id == current_user.id
```

Comment:
This should be done on the controllers query where you filter out blog ownership of the signed in user.